### ISSUE TITLE
Support PG 15

### ISSUE THREAD

**Issue by lelit** [lelit/pglast#110]
It seems likely that `libpg_query` [will cover PG 15](https://github.com/pganalyze/libpg_query/pull/167) sooner than expected...

**Comment by lelit** [comment]
The new `v5` branch uses the _in development_ [libpg_query PG 15](https://github.com/pganalyze/libpg_query/pull/167) .

**Comment by df7cb** [comment]
There's a slight regression in the statements renderer in v5 (compared to v3, did not try v4):
```
>>> pglast.prettify('create cast (foo as bar) without function')
'CREATE CAST (foo AS bar)WITHOUT FUNCTION'
```

Previously, there was a space before the `WITHOUT`.

**Comment by lelit** [comment]
Thanks for reporting, and yes, it affects v4 as well.

**Comment by lelit** [comment]
I fixed that regression (and a few other similar ones) and released v4.1 and v5.0.dev0.

**Comment by lelit** [comment]
This is v5.0 final.
